### PR TITLE
[board] add support for Parrot Disco plane

### PR DESCRIPTION
**Pull request by gautierhattenberger**
- mostly based on Bebop2
- add PWM output and SBUS input on Linux (based on ArduPilot work)
- some changes to the Linux serial port lib to handle arbitrary baudrate
  (this code could be improved and factorized in the future)
- all sensors are working, the video system is not tested

### CODE REVIEW

**Comment by fvantienen** [comment]
Looks good to me

File: sw/airborne/arch/linux/mcu_periph/pwm_sysfs.c [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  va_end(args);
+
+  if (ret < 1) {
+    return -errno_bkp;
+  }
+
+  return ret;
+}
+
+/*
```
Comment (flixr) [inline sw/airborne/arch/linux/mcu_periph/pwm_sysfs.c]: whats up with this commented out code?

Comment (gautierhattenberger) [inline sw/airborne/arch/linux/mcu_periph/pwm_sysfs.c]: original code had getter functions for the different parameters (by reading values in the system files). I commented this part to remove an unused warning and forgot it. If you think it is not useful for future improvements (checking status) I can remove it.

Comment (flixr) [inline sw/airborne/arch/linux/mcu_periph/pwm_sysfs.c]: Then I would rather use a pragma to silence the warning and keep the code there...

**Comment by gautierhattenberger** [comment]
done

**Pull request merged**